rate: 20
# initial coordinates (robot's, not LIDAR's coords!)
start_yellow: [2.75, 0.94, 3.14]
start_blue: [0.25, 0.94, 0]

# LIDAR scanner coordinates in robot frame
lidar_x: 0
lidar_y: 0
lidar_a: 0

beacons_radius: 0.048
beacon_range: 0.2
min_range: 0.05
alpha: 1
min_point_per_beacon: 3

#Table and beacon parameters
world_x: 3
world_y: 2
world_border: 0.022
beac_l: 0.1
beac_border: 0.022
scan_offset: 0.018
beac_radius: 0.048

# Particle filter parameters
particles_num: 500
distance_noise: 0.002
angle_noise: 0.0035
max_dist: 3700
distance_noise_1_beacon: 0.0133
angle_noise_1_beacon: 0.02
sigma_phi: 0.03
min_sin: 0.4
sigma_r: 0.015
num_seeing_beacons: 3
particles_num_from_measurement_model: 5

